examples/sn-react-component-docs/stories/universal-header.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { UniversalHeader } from '@sensenet/universal-header-react'
```

Comment:
package name and its folder should follow a pattern where name is a placeholder:

folder: packages/sn-name
package: @sensenet/name